### PR TITLE
fix (docs): missing link to guide on how to create a provider

### PR DESCRIPTION
**Pull request by zakariaelas**
Hey guys, really cool project!

I was reading the docs and came across a missing link on the [Integrations Store](https://docs.novu.co/platform/integrations) page. I think the "here" should point to the guide on how to create a custom provider.

PR adds just a tiny fix 👍

### CODE REVIEW

File: docs/docs/platform/integrations.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ For email providers, you will be required to add the following data:
 
 ## Missing a provider?
 
-Novu is an open-source platform, meaning that if you are missing a particular provider you can create the provider best on the guide specified here.
+Novu is an open-source platform, meaning that if you are missing a particular provider you can create the provider best on the guide specified [here](/community/create-provider).
```
Comment (ainouzgali) [inline docs/docs/platform/integrations.md:26]:
```suggestion
Novu is an open-source platform, meaning that if you are missing a particular provider you can create the provider using the guide specified [here](/community/create-provider).
```

Comment (ainouzgali) [reply docs/docs/platform/integrations.md:26]: Also noticed this weird phrasing thanks to you :) 
What do you think? Does this sound better?

**Review by ainouzgali — COMMENTED** [review]
This is great @zakariaelas ! Thank you so much for noticing and taking the time to change it

File: docs/docs/platform/integrations.md (hunk 1 above)
Comment (zakariaelas) [reply docs/docs/platform/integrations.md]: Nice catch, I think this is better 😄

**Review by ainouzgali — APPROVED** [review]
🤗 Thanks!

**Pull request merged**